src/index.php

```diff
@@ -52,11 +52,12 @@
                             <label class="small blocklabel">You see this window because you or someone set an PIN in this device and choose it as default login method. To switch account or disable PIN, please press the red button below. To use username/password to login only this time, close this window by pressing the 'X' at top-right corner.</label>
                             <label class="small textred blocklabel">Closing this window only let you use username/password to login this time. PIN will still be chosen as default method in future. Press red button below if you want to disable current PIN.</label>
                         </div>
-                </div>
-                <div class="modal-footer">
-                    <p class="collapse" id="pinerrorhint">PIN ERROR, try again.</p>
-                    <button type="button" class="btn btn-danger" data-dismiss="modal">Delete PIN</button>
-                    <input type="submit" class="btn btn-primary" id="pinlogin" value="Login" /></form>
+                        <div class="modal-footer">
```

Comment:
`modal-footer` should be on the same level as `modal-body` so i guess the correct solution would be to make the `form`-tag embrace all the elements of `modal-content`